packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/SynapseTableCell.tsx

```diff
@@ -210,43 +206,7 @@ export function SynapseTableCell(props: SynapseTableCellProps) {
       return <p className={isBold}>{formatDate(dayjs(Number(columnValue)))}</p>
 
     case ColumnTypeEnum.USERID:
-      if (
-        Object.prototype.hasOwnProperty.call(mapUserIdToHeader, columnValue)
-      ) {
-        const cachedData = mapUserIdToHeader[columnValue]
-        const { ownerId, userName } = cachedData
-        if ('isIndividual' in cachedData) {
-          // is UserGroupHeader
-          const icon: IconName =
-            userName === AUTHENTICATED_USERS ? 'public' : 'people'
-          if (userName === AUTHENTICATED_USERS) {
-            return (
-              <span>
-                <IconSvg icon={icon} /> All registered Synapse users
-              </span>
-            )
-          }
-          return (
-            <Link
-              target="_blank"
-              rel="noopener noreferrer"
-              href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Team:${ownerId}`}
-            >
-              <IconSvg icon={icon} /> {userName}
-            </Link>
-          )
-        } else {
-          // isUserCard
-          return (
-            <UserCard
-              userProfile={cachedData}
-              preSignedURL={cachedData.clientPreSignedURL}
-              size={'SMALL USER CARD'}
-            />
-          )
-        }
-      }
-      break
+      return <UserOrTeamBadge principalId={columnValue} />
```

Comment:
I think this fixes https://sagebionetworks.jira.com/browse/PORTALS-2403, could you verify?

Reply:
Oh wait, UserIdList would also need to use UserOrTeamBadge.

Reply:
I went ahead and updated UserIdList in [this commit](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/428/commits/9b1e7ea22dff3f0d9f30d21672700880d2497391)